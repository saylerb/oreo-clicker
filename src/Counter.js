import React from "react";

const Counter = () => <div data-testid="current-count">Count: 0</div>;

export default Counter;
