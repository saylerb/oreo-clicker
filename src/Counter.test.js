import React from "react";
import { create } from "react-test-renderer";
import Counter from "./Counter";

test("the counter starts at zero", () => {
  const counter = create(<Counter />);

  expect(counter.toJSON()).toMatchSnapshot();
});
