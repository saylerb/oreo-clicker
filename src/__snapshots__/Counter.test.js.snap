// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the counter starts at zero 1`] = `
<div>
  Count: 0
</div>
`;
